backend/authentication/templates/pwreset_email.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Password Reset</title>
+</head>
+<body>
+    <p>Hi {{username}},</p>
+    <p>We have received a request to reset your password. To proceed with the password reset, please click on the link below:</p>
+    <p><a href={{pwreset_link}}>Reset Password</a></p>
+    <p>If you did not request a password reset, please ignore this email.</p>

```

Comment:
We could do "If you did not request a password reset, then it's safe to ignore this email."